[2024-05-28T03:52:04.500+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-28T03:52:04.552+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: download_files_batch.download_and_upload_task_0 scheduled__2024-05-28T03:50:00+00:00 [queued]>
[2024-05-28T03:52:04.561+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: download_files_batch.download_and_upload_task_0 scheduled__2024-05-28T03:50:00+00:00 [queued]>
[2024-05-28T03:52:04.562+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-05-28T03:52:04.597+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): download_and_upload_task_0> on 2024-05-28 03:50:00+00:00
[2024-05-28T03:52:04.600+0000] {standard_task_runner.py:63} INFO - Started process 199 to run task
[2024-05-28T03:52:04.601+0000] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'download_files_batch', 'download_and_upload_task_0', 'scheduled__2024-05-28T03:50:00+00:00', '--job-id', '46', '--raw', '--subdir', 'DAGS_FOLDER/download-files_batch.py', '--cfg-path', '/tmp/tmptz3z5lz8']
[2024-05-28T03:52:04.603+0000] {standard_task_runner.py:91} INFO - Job 46: Subtask download_and_upload_task_0
[2024-05-28T03:52:04.719+0000] {task_command.py:426} INFO - Running <TaskInstance: download_files_batch.download_and_upload_task_0 scheduled__2024-05-28T03:50:00+00:00 [running]> on host 7f87f2070cf0
[2024-05-28T03:52:04.834+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='download_files_batch' AIRFLOW_CTX_TASK_ID='download_and_upload_task_0' AIRFLOW_CTX_EXECUTION_DATE='2024-05-28T03:50:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-05-28T03:50:00+00:00'
[2024-05-28T03:52:04.835+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-28T03:52:04.849+0000] {logging_mixin.py:188} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/utils/context.py:320 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2024-05-28T03:52:53.977+0000] {logging_mixin.py:188} INFO - File downloaded successfully from https://covid.ourworldindata.org/data/owid-covid-data.csv!
[2024-05-28T03:52:55.462+0000] {logging_mixin.py:188} INFO - Error uploading file our-world-in-data-covid19.csv to MinIO: S3 operation failed; code: AccessDenied, message: Access Denied., resource: /covid19dataset/our-world-in-data-covid19.csv, request_id: 17D38AB0DFE8C467, host_id: dd9025bab4ad464b049177c95eb6ebf374d3b3fd1af9251148b658df7ac2e3e8, bucket_name: covid19dataset, object_name: our-world-in-data-covid19.csv
[2024-05-28T03:52:55.462+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-28T03:52:55.462+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-28T03:52:55.472+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=download_files_batch, task_id=download_and_upload_task_0, run_id=scheduled__2024-05-28T03:50:00+00:00, execution_date=20240528T035000, start_date=20240528T035204, end_date=20240528T035255
[2024-05-28T03:52:55.514+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-28T03:52:55.528+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-28T03:52:55.529+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
